component/breakdown.js

```diff
@@ -517,8 +517,7 @@ export default class Breakdown extends Component {
           <GridItem columnSpan={4} className="cohort tolerated">
             <Icon
               className="icon"
-              sizeType={Icon.SIZE_TYPE.NORMAL}
-              type={Icon.TYPE.INTERFACE__STATE__WARNING}
+              type={Icon.TYPE.INTERFACE__STATE__WARNING__SIZE_16}
```

Comment:
I don't think we need the `__SIZE_16` suffix here since 16px is the default for icons

Reply:
will fix.